package/MDAnalysis/core/universe.py

```diff
@@ -808,7 +808,7 @@ def add_TopologyAttr(self, topologyattr, values=None):
 
         >>> u.add_TopologyAttr('bfactors')
         >>> u.atoms.bfactors
```

Comment:
**This is unrelated to your changes, therefore feel free to ignore the following comment if you don't want to address it in this PR or at all.**

I can see from the log that there are some warnings because `bfactors` has been deprecated (and will be removed in 3.0).

```
/home/runner/work/mdanalysis/mdanalysis/package/MDAnalysis/core/universe.py:828: DeprecationWarning: The bfactor topology attribute is only provided as an alias to the tempfactor attribute. It will be removed in 3.0. Please use the tempfactor attribute instead.
```

If you want/have time you could fix this warning as well (which will become an error down the line) by replacing `bfactor`  with `tempfactor` (and `bfactors` with `tempfactors`).